aspnetcore/blazor/file-uploads.md

```diff
@@ -548,6 +548,52 @@ In Blazor WebAssembly, file data is streamed directly into the .NET code within
 
 :::zone-end
 
+## Upload image preview
```

Comment:
Do we need to copy this below for .NET 6.0?

Reply:
Yes, but I'll take care of it on Monday morning.

Does this PR cover #26276?